build.sh

```diff
@@ -453,39 +451,41 @@ if (( ${NUMARGS} == 0 )) || hasArg libraft || hasArg docs || hasArg tests || has
       MSG="${MSG}<br/>parallel setting: $PARALLEL_LEVEL"
       MSG="${MSG}<br/>parallel build time: $compile_total seconds"
       if [[ -f "${LIBRAFT_BUILD_DIR}/libraft.so" ]]; then
-          LIBRAFT_FS=$(ls -lh ${LIBRAFT_BUILD_DIR}/libraft.so | awk '{print $5}')
+          LIBRAFT_FS=$(find "${LIBRAFT_BUILD_DIR}" -name libraft.so -printf '%s')
```

Comment:
I recall seeing @divyegala change a similar thing in `cuml` from `ls -lh` to `stat -c`: https://github.com/rapidsai/cuml/pull/6780#discussion_r2138347882

and adding in the file size.

Would you consider doing that here too, since this is touching this line anyway?

Reply:
Yeah, `find` is doing the same thing here, but printing out the number in bytes -- I can add the `awk` magic to format it as `%.2f MB`

Reply:
ok cool, thanks for letting me expand the scope of this a tiny bit 😬